app.py

```diff
@@ -37,4 +38,5 @@ def move():
 
 if __name__ == "__main__":
     print("Starting server...")
-    app.run(port=8080, debug=True)
+    port = int(os.environ.get("PORT", 8080))
+    app.run(host='0.0.0.0', port=port, debug=True)
```

Comment:
debug should be off for prod version. is probably better to just use the FLASK_DEBUG environment variable when running locally.